modules/rtm/NAMESPACE

```diff
@@ -39,6 +39,7 @@ export(params.prospect4)
 export(params.prospect5)
 export(params.prospect5b)
 export(params2edr)
+export(plot.spectra)
```

Comment:
Suggested this on Slack, but re-iterating here:

Not sure why this S3 method is being recognized all of a sudden. I would try one or more of the following in the `plot.spectra` definition:
* `#' @importFrom graphics plot` -- maybe the issue is that it's not finding the `plot` S3 generic in the PEcAnRTM NAMESPACE, and that's why it's not recognizing it as an S3 method. If you do this, you may also have to add `graphics` to the `Imports:` in the DESCRIPTION.
* `#' @method plot spectra` -- explicitly declare it as a method. I would only do this if the previous suggestion fails -- in general, `roxygen` should be smart enough to figure this out.

Reply:
Done